src/main/java/com/sergiuszg/githubdetailsservice/configuration/CustomRequestInterceptor.java

```diff
@@ -0,0 +1,13 @@
+package com.sergiuszg.githubdetailsservice.configuration;
+
+import feign.RequestInterceptor;
+import feign.RequestTemplate;
+
+public class CustomRequestInterceptor implements RequestInterceptor {
+
+    @Override
+    public void apply(RequestTemplate template) {
```

Comment:
Prośba o doczytanie dla ktorych requestow to sie doda czyt wszystkich, ktore beda w aplikacji wysylane feign clientem czy moze tylko tych dla danego clienta.